File: Python/stack.py
array = []
for i in range(5):
    take_input = input("Value: ")
    array.append(take_input)

while True:
    remove_array = input("Remove Value? (y/n): ")
    if remove_array.lower() == "y": 
        if not array:
            print("Stack is empty")
        else:
            removed_value = array.pop()
            print("Removed Value: ", removed_value)
            print("Current Stack: ", array)


    elif remove_array.lower() == 'n':break
    else: print("Invalid Input. Please Enter 'y' or 'n'")

print("Final Array: ",array)
print("The time complexity of this algorith is: O(1) Constant TIme")

File: Go/stack.go
package main

import (
	"fmt"
)

func stack(array [5]int) {
	fmt.Println("Stack here")
	for i := 0; i <= len(array); i++ {
		fmt.Print("Remove Value? (y/n):")
		var input string
		fmt.Scan(&input)
		fmt.Print(input)

		fmt.Scan(input)
	}
}

func main() {
	stack_array := [5]int{1, 2, 3, 4, 5}
	stack(stack_array)

}
